GVFS/GVFS.UnitTests.Windows/Windows/Mock/MockVirtualizationInstance.cs

```diff
@@ -174,12 +149,61 @@ public void WaitForCreateWriteBuffer()
             this.waitForCreateWriteBuffer.WaitOne();
         }
 
+        public HResult CompleteCommand(int commandId, NotificationType newNotificationMask)
+        {
+            this.commandCompleted.Set();
+            return HResult.Ok;
+        }
+
+        public HResult CompleteCommand(int commandId, IDirectoryEnumerationResults results)
+        {
+            this.commandCompleted.Set();
+            return HResult.Ok;
+        }
+
+        HResult IVirtualizationInstance.CompleteCommand(int commandId, HResult completionResult)
+        {
+            this.commandCompleted.Set();
+            this.CompletionResult = completionResult;
+            return HResult.Ok;
+        }
+
+        public HResult CompleteCommand(int commandId)
+        {
+            this.commandCompleted.Set();
+            return HResult.Ok;
+        }
+
         public void Dispose()
         {
             this.Dispose(true);
             GC.SuppressFinalize(this);
         }
 
+        public HResult WriteFileData(Guid dataStreamId, IWriteBuffer buffer, ulong byteOffset, uint length)
+        {
+            return this.WriteFileReturnResult;
+        }
+
+        IWriteBuffer IVirtualizationInstance.CreateWriteBuffer(ulong byteOffset, uint length, out ulong alignedByteOffset, out uint alignedLength)

```

Comment:
Where is this version of `CreateWriteBuffer` called? (If it's not getting called can we just have it throw `NotImplementedException`)